README.md

```diff
@@ -228,7 +242,7 @@ See the [contributing guide](CONTRIBUTING.md) for general guidelines.
 make test
 ```
 
-### Running integration tests
+### Running integration tests agains a real OCM environment
```

Comment:
typo: against vs sgains (note the missing T at the end)

Reply:
thanks